qcsrc/deathmatch_settings.qc

```diff
@@ -0,0 +1,208 @@
+//
+// deathmatch_settings.qc -- Centralized location for defining which deathmatch
```

Comment:
For most of the functions in here, we can use #define macros instead since they are returning comparison values. This will keep our progs a little smaller in size and provide some slight speedups which may prove beneficial for very low end machines.